src/client/pages/Checkout/CheckoutDetails/CheckoutDetails.tsx

```diff
@@ -2,21 +2,40 @@ import React from 'react'
 import { useTextKeys } from 'utils/textKeys'
 import { useQuoteCartIdFromUrl } from 'utils/hooks/useQuoteCartIdFromUrl'
 import { useCurrentLocale } from 'l10n/useCurrentLocale'
+import { useQuoteCartData } from 'utils/hooks/useQuoteCartData'
 import { CheckoutPageWrapper } from '../shared/CheckoutPageWrapper'
 import { Footer } from '../shared/Footer'
+import { StartDate } from '../../Offer/Introduction/Sidebar/StartDate'
+import { YourPlan } from './components/YourPlan/YourPlan'
+import { QuoteDetails } from './components/QuoteDetails/QuoteDetails'
+import { PageSection } from './components/PageSection'
+import { SubSection } from './components/SubSection'
 
 export const CheckoutDetails = () => {
   const textKeys = useTextKeys()
   const { path: localePath } = useCurrentLocale()
   const { quoteCartId } = useQuoteCartIdFromUrl()
-
+  const data = useQuoteCartData()
+  if (!data) {
+    return null
+  }
```

Comment:
How does this look for users?

Reply:
Will be handled in the error + loading state PR.